apps/docs/src/basics/typography.md

```diff
@@ -0,0 +1,350 @@
+# Typography
+
+::: danger
+Still content to create!
+:::
+
+The onyx font system includes three different font families that can be used throughout your application, pending on the usecase. It contains several headline styles as well as paragraph styles and mono space styles.
+
+::: info
+The font families that can be used with onyx are:
+
+- [Source Sans 3](https://fonts.google.com/specimen/Source+Sans+3)
+- [Source Code Pro](https://fonts.google.com/specimen/Source+Code+Pro?query=source)
+- [Lidl Font Pro](https://www.brand.lidl/brandelements/brandelements/view/id/664#headline_2486)
+- [Kaufland Regular](https://google.com)
```

Comment:
Until we have a decision to make lidl/kaufland etc. theme public I'd like to remove it here

Reply:
Also remove all content below that references lidl or kaufland

Reply:
I would write "lidl and kaufland font to come later". I would not delete it as a whole